shanoir-ng-users/src/main/java/org/shanoir/ng/ShanoirUsersManagement.java

```diff
@@ -207,4 +218,30 @@ private UserRepresentation getUserRepresentation(final User user) {
 		return userRepresentation;
 	}
 
+	/**
+	 * Set up the email ${service-account.user.email}
+	 * of the keycloak user ${service-account.user.name} ('service-account-service-account')
+	 * associated with the keycloak client 'service-account'
+	 *
+	 * See service-account.user.* application properties
+	 */
+	private void setVIPServiceAccountEmail(){
+
+		final List<UserRepresentation> userRepresentationList = keycloak.realm(keycloakRealm).users().searchByUsername(this.vipSrvUsername, true);
+		if (userRepresentationList == null || userRepresentationList.isEmpty()) {
```

Comment:
Small doubt here. Shouldn't we create this user if it does not exists ?

Reply:
Hi @jcomedouteau,
This user is automatically created with service-account client.
If it does not exists, that mean the service-account is not set.
So i think creating it wouldn't have a lot of meaning here.